### PR TITLE
Add acceptance

### PR DESCRIPTION
**Pull request by martialblog**
Hi, 

I added some simple acceptance tests. Right now they just check if everything gets installed properly, but they can now be easily extended. Took the liberty of adding the new tests to the Travis CI configuration. 

While testing I also found and fixed an issue with the Debian installation. 

Let me know what you think.
Cheers

### CODE REVIEW

**Comment by martialblog** [comment]
Had to adjust some stuff in the Travis config. Should work now

File: spec/acceptance/nodesets/docker/centos-7.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ HOSTS:
     docker_preserve_image: true
     docker_cmd: '["/usr/sbin/init"]'
     docker_image_commands:
-      - 'yum install -y crontabs initscripts iproute openssl sysvinit-tools tar wget which ss'
+      - 'yum install -y git crontabs initscripts iproute openssl sysvinit-tools tar wget which ss git'
```
Comment (bastelfreak) [inline spec/acceptance/nodesets/docker/centos-7.yml]: git is twice here

File: spec/acceptance/nodesets/docker/debian-8.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ HOSTS:
     docker_cmd: '["/sbin/init"]'
     docker_image_commands:
       - 'echo deb http://ftp.debian.org/debian jessie-backports main >> /etc/apt/sources.list'
-      - 'apt-get update && apt-get install -y cron locales-all net-tools wget'
+      - 'apt-get update && apt-get install -y cron git locales-all net-tools wget git'
```
Comment (bastelfreak) [inline spec/acceptance/nodesets/docker/debian-8.yml]: git is twice here

**Comment by bastelfreak** [comment]
Hi @martialblog, thanks for the PR! the .travis.yml is generated by our [modulesync_config](https://github.com/voxpupuli/modulesync_config). Please add the beaker tests also to the .sync.yml, otherwise the next modulesync will purge it from .travis.yml. [Here](https://github.com/voxpupuli/puppet-corosync/blob/master/.sync.yml) is an example from our corosync module.

All the nodesets come directly from [modulesync_config](https://github.com/voxpupuli/modulesync_config/tree/master/moduleroot/spec/acceptance/nodesets) too. Most of our tests don't need git. Do you see any easy way to install git without modifying the nodsets? If not, please provide a PR to modulesync_config as well with the git packages in the nodesets.

**Comment by martialblog** [comment]
Hey, I'll have a look at it. Should be possible using wget...

**Comment by martialblog** [comment]
Hey, I added the Beaker tests to the _.sync.yml_ and adjusted the tests to they don't need git.

**Comment by bastelfreak** [comment]
thanks!

**Pull request merged**